### ISSUE TITLE
Is it possible to come back to the aspect of 0.19?

### ISSUE THREAD

**Issue by JavierLopezT** [BookStackApp/BookStack#1843]
Hello. I had version 0.19 and I liked very much the aspect of it, very vivid colors. I have upgraded to the last version and it looks worse for me, because the colors are darker and the book icon is different. Is there any way of getting back the colors/aspect of 0.19 while maintaining the new features of 0.27.5? 

Thank you very much

**Comment by ssddanbrown** [comment]
Hi @JavierLopezT,

Version 0.19 to 0.27 is quite a leap.

We recently tweaked the default colors to be a little darker by default for accessibility purposes. It's worth looking at this issue: #1380

To summarise, In the latest release version of BookStack, The theme colors used can easily be overridden using some custom CSS as a setting. In BookStack v0.28, which I'm hoping to release next weekend, #1723 will be implemented which allows these colors to be easily changed in the settings area in an easier manner.

I'm not sure about the icon though, I did not think we had changed the book icon.
